### PR TITLE
Dockerfile and workflow for dockerHub

### PR DESCRIPTION
**Pull request by Cristian-Baeza**
- Added a dockerfile
- Added a workflow to update remote image on every merge and push to master

### CODE REVIEW

File: .github/workflows/docker-image.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Build and Push Docker Image
+        uses: mr-smithers-excellent/docker-build-push@v5
+        with:
+          image: cbaeza8/ruby-test-repo
```
Comment (td-extreme) [inline .github/workflows/docker-image.yml:18]: Where did this image name come from?

File: .github/workflows/docker-image.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+          tags: latest
+          registry: docker.io
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
```
Comment (td-extreme) [inline .github/workflows/docker-image.yml:22]: How / Where are these `secrets` defined used?

**Review by td-extreme — COMMENTED** [review]
Dockerfile looks fine. 
I have not used github workflows before and was just curious about where some of the values came from?
Can you provide a link to the github workflows guide you used?

**Comment by td-extreme** [comment]
Can you add an update to the README with at information about how a new deploy to docker hub happens, and what docker command to run to play the game?

**Comment by Cristian-Baeza** [comment]
> Dockerfile looks fine. I have not used github workflows before and was just curious about where some of the values came from? Can you provide a link to the github workflows guide you used?

Yup, I used this: 
https://github.com/marketplace/actions/docker-build-push-action

**Review by td-extreme — APPROVED** [review]

**Pull request merged**